hbase-client/src/main/java/org/apache/hadoop/hbase/client/ScannerCallableWithReplicas.java

```diff
@@ -262,6 +262,7 @@ public Result[] call(int timeout) throws IOException {
     throw new IOException("Imposible? Arrive at an unreachable line...");
   }
 
+  @SuppressWarnings("FutureReturnValueIgnored")
```

Comment:
Suppressed here because scheduling the callable returns a future we don't care about.